graphql/schema.graphql

```diff
@@ -5,15 +5,23 @@ scalar Date @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\Date")
 scalar DateTime
     @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")
 
```

Comment:
did you run that and saw all queries, because right now you override the query type in every model so you will end up with the last one only, so to fix this you need to do the following
1- create basic Query and Mutation types in `schema.graphql`
```
type Query
type Mutation
```
2- then for every model make sure you extend the `Query` and `Mutation` types  example
```
extend Query{
 // put model your queries here
}
extend type Mutation{
 // put model mutations here 
}
```